packages/gitbook/src/components/Search/server-actions.tsx

```diff
@@ -48,6 +49,7 @@ export interface ComputedPageResult {
 
     /** When part of a multi-spaces search, the title of the space */
     spaceTitle?: string;
+    section?: SiteSection;
```

Comment:
We should not pass all of these data to the client side.

Instead we should merge both `spaceTitle` and `section` as a `breadcrumbs` property here

Something like:

```
breadcrumbs?: Array<{ icon?: IconName; label: string; >
```